rakelib/artifacts.rake

```diff
@@ -437,6 +496,12 @@ namespace "artifact" do
       excluder = oss_excluder
     end
 
+    if bundle_jdk
+      suffix += "-with-jdk-x86_64"
```

Comment:
Is the intention to diverge from elasticsearch and have `-with-jdk-{arch}` and `-no-jdk`, or have `{arch}` and `-no-jdk`?